File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Sak.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class Sak {

    private X001Anmodning anmodning;

    private Kontekst kontekst;

    private X006FjernInstitusjon fjerninstitusjon;

    private Ugyldiggjoere ugyldiggjoere;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Sak.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class Sak(
    var anmodning: X001Anmodning? = null,
    var kontekst: Kontekst? = null,
    var fjerninstitusjon: X006FjernInstitusjon? = null,
    var ugyldiggjoere: Ugyldiggjoere? = null
)
